PostHog/Replay/UIView+Util.swift

```diff
@@ -44,5 +44,10 @@
 
             return image
         }
+
+        // you need this because of SwiftUI otherwise the coordinates always zeroed for some reason
```

Comment:
took me a few hours and a bit of digging to find this, otherwise x, and y are always zeroed for SwiftUI componenets

Reply:
"This one trick will save you 4 hours building mobile session replay" - A buzzfeed clickbait article by PostHog